scripts/map.js

```diff
@@ -0,0 +1,69 @@
+jQuery(() => {
+  // Asynchronously Load the map API
+  const script = document.createElement('script');
+  script.src = '//maps.googleapis.com/maps/api/js?sensor=false&callback=initialize';
+  document.body.appendChild(script);
+});
+
+// eslint-disable-next-line no-unused-vars
+function initialize() {
+  const bounds = new google.maps.LatLngBounds();
+  const mapOptions = {
+    mapTypeId: 'roadmap',
+  };
+
+    // Display a map on the page
+  const map = new google.maps.Map(document.getElementById('map_canvas'), mapOptions);
+  map.setTilt(45);
+
+  // Multiple Markers
+  const markers = [
```

Comment:
This might be better as an array of objects, like so:
```
const markers = [
  {name: 'The Pitts Family Home', latitude: 44.345, longitude: -79.456},
  ...
];
```

Just for clarity of what each part represents, and then it makes accessing the parts of the markers more clear.